src/lib/user-features.ts

```diff
@@ -318,10 +332,10 @@ const shouldNotBeShownSupportMessaging = (): boolean =>
     which this function is dependent on.
 */
 
-const shouldHideSupportMessaging = (): boolean =>
+const shouldHideSupportMessagingOkta = async (): Promise<boolean> =>
```

Comment:
In a lot of the cases in Frontend we created `Okta` variants of methods when we needed access to the old pre-okta method aswell as the new Okta method, but if you're able to just refactor the old method to work for Okta I don't think theres any need to suffix methods with `Okta`